### DISCUSSION TITLE
I have problems with direct IP when using AGH in Windows 11 22H2

### DISCUSSION THREAD

**Discussion by DandelionSprout** [AdguardTeam/AdGuardHome#4949] [Q&A]
To ensure I've not become paranoid before submitting a proper issue report:

Do anyone else than me have problems with direct IP connections to AGH no longer working (or are so ultra-slow that connections time out) after updating the server OS to Windows 11 22H2, both with AGH 0.107.13 and 0.108.0-b.15? Apparently my server has been down for 4 days except for DoH.

From what I can tell on my end:
* `netstat -an` shows `0.0.0.0:53` and `[::]:53` as `LISTENING` on TCP, and as `*:*` on UDP.
* `AdGuardHomeLogFile.txt` shows `[info] Listening to udp://[::]:53`, but strangely enough does **not** show `[info] Listening to udp://127.0.0.1:53`. There seem to be no mention of listeners being closed.
* "Windows Defender Firewall with Advanced Settings" claims that port 53 has been completely opened, though it has been mistaken before.
* Port 53 seems to be forwarded on both IPv4 and IPv6 on my ASUS router.
* `.\AdGuardHome.exe --service restart` seems to make it work for between 2 and 60 seconds again, after which it no longer works again.
* The AdGuard for Android logger shows `04:47:43.039 [AsyncTask #7] WARN  com.adguard.android.filtering.dns.c - Failed to test 192.168.1.188 java.lang.IllegalArgumentException: Request timed out`
* My Nintendo Switch can't connect either, usually with `Error Code: 2110-3127 (...) Could not perform DNS name resolution`
* Trying to connect to `127.0.0.1` as DNS from the server PC doesn't work either, remarkably enough.

**Comment by ainar-g** [comment]
Sorry for the late response.  My first instinct would be make sure that nothing else is interfering, including rechecking Windows Defender.  Also, if you have other network filtering / proxy / VPN software, they could interfere with AdGuard Home as well.  I don't know what the Windows equivalent of `sudo lsof -i :53` is, but that could help.

**Reply by DandelionSprout** [reply]
I should've probably mentioned in this thread too that I moved the focus over to https://github.com/AdguardTeam/AdGuardHome/issues/4975, which was fixed a week later or so. 😅
